### DISCUSSION TITLE
World 에서 PostProcessing 질문

### DISCUSSION THREAD

**Discussion by peterkangdev** [naverz/zepeto-studio-kor#101] [5. Q&A]
World 에서 Post Processing 이 안되는 것 같은데.. 제가 잘못하여 안되는 것인지 지원하지 않는 것인지 궁금합니다.

**Comment by cheolhui-lee** [comment]
안녕하세요.
아쉽지만 현재 제페토에서는 지원하고 있지 않습니다.
지원 여부에 대해 내부 검토를 계속 진행 중인 점 참고해주세요.
